codemon/CodemonReg.py

```diff
@@ -6,22 +6,31 @@
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 
 def codemonReg():
-    print(ROOT_DIR)
-    path = os.path.join(ROOT_DIR, "codemon")
+    #print(ROOT_DIR)
+    prefix = '.' if os.name != 'nt' else ''
+    file_name = prefix + "codemon"
+    path = os.path.join(ROOT_DIR, file_name)
     if not os.path.exists(path):
         os.makedirs(path)
-        FILE_ATTRIBUTE_HIDDEN = 0x02
-        ret = ctypes.windll.kernel32.SetFileAttributesW(path, FILE_ATTRIBUTE_HIDDEN)
-    meta = ""
-    name = input("Name : ")
-    cf_user = input("CF Username : ")
-    cf_pass = input("CF Password : ")
-    git_user = input("GitHub Username : ")
-    git_pass = input("GitHub Password : ")
-    meta += name + "\n" + cf_user + "\n" + cf_pass + "\n" + git_user + "\n" + git_pass
-    full_filename = os.path.join(ROOT_DIR, "codemon", "meta.txt")
+        if os.name == 'nt':
+            FILE_ATTRIBUTE_HIDDEN = 0x02
+            ret = ctypes.windll.kernel32.SetFileAttributesW(path, FILE_ATTRIBUTE_HIDDEN)
```

Comment:
Please explain what this piece of code is supposed to do? And how is the `ret` is used ?

Reply:
In windows, we can't create Hidden directories by simply adding '.' prefix. So, this piece of code is changing the attribute of file to make it hidden.
'ret' stores the result of that process, if it was successful in doing so or not.